File: llm_generated_code/ChatGPT/Python/Level1/Q37.py
def min_cost_to_make_good(s):
    cost = 0
    i = 0
    while i < len(s):
        if i + 1 < len(s) and s[i] == s[i + 1]:
            cost += 1
            i += 2
        else:
            i += 1
    return cost

t = int(input())
results = []
for _ in range(t):
    s = input().strip()
    results.append(min_cost_to_make_good(s))

for result in results:
    print(result)

File: llm_generated_code/ChatGPT/CPP/Level1/Q37.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    int t;
    cin >> t;
    
    while (t--) {
        string s;
        cin >> s;
        
        int n = s.size();
        int cost = 0;
        
        for (int i = 0; i < n - 1; i++) {
            if (s[i] == s[i + 1]) {
                cost++;
                i++;
            }
        }
        
        cout << cost << endl;
    }
    
    return 0;
}
